src/mds/topology/topology_metric.cpp

```diff
@@ -232,6 +234,29 @@ void TopologyMetricService::UpdateTopologyMetrics() {
         }
     }
 
+    // cluster io
+    uint64_t clusterReadIOPS = 0, clusterWriteIOPS = 0,
+        clusterReadRate = 0, clusterWriteRate = 0;
+    for (auto metricIter = gChunkServerMetrics.begin();
```

Comment:
The cluster read/write iops and rate only need add all the pools' related metric value( https://github.com/opencurve/curve/pull/2176)

Reply:
fix